src/python/pants/backend/native/config/environment.py

```diff
@@ -42,34 +42,54 @@ def resolve_platform_specific(self, platform_specific_funs):
     return fun_for_platform()
 
 
-class Executable(object):
+# NB: @abstractproperty requires inheriting from AbstractClass to work!
```

Comment:
This is both untrue and doesn't stand the novelty test. Using AbstractClass is a convenient way to properly support `@abstractproperty` but not the only way, and we do this in many places in the codebase, so the revelation - though personally novel is not so generally and so probably not worth a comment.

Reply:
This was a note to myself which I was planning to remove, and this was a note that I was going to word as "inheriting from AbstractClass, or an equivalent method to set the metaclass", then decided I probably didn't need to go that in depth for now. I am aware this is being presented as code for review, and I agree that this shouldn't be in there, but as mentioned in the other comment, since I "need" travis to run tests, this all gets flattened.

I suppose in the future I could keep TODOs in another file, but that's not ideal and I would prefer to make the local testing situation better to avoid pushing code to a PR that I intend to remove.

Reply:
This note has been removed.